go.mod

```diff
@@ -156,7 +158,9 @@ require (
 replace (
 	// TODO: fix keyring upstream
 	github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.8
+
+	// TODO: might use crypto-org repo as the refrence path
+	github.com/cosmos/cosmos-sdk => github.com/jayt106/cosmos-sdk v0.45.8-db-opts
```

Comment:
Should create the PR to upstream?

Reply:
I will create it, but not sure the SDK team will want to change it because the `tm-db` version changed.

Reply:
Yeah, usually we create the PR to main first, then try to backport